plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -466,6 +467,10 @@ public void setTableComment(ConnectorSession session, ConnectorTableHandle table
     public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, Optional<ConnectorTableLayout> layout, RetryMode retryMode)
     {
         verify(transaction == null, "transaction already set");
+        String schemaName = tableMetadata.getTable().getSchemaName();
```

Comment:
Let's swap L469 and L479.

Reply:
sorry,  I would like to confirm， L479 or L470?